vault/.obsidian/plugins/obsidian-git/config.mobile.json

```
@@ -0,0 +1,17 @@
+{
+  "autoPullInterval": 60,
```

Biome (2.5.7)

[error] The key autoPullInterval was already declared.

(lint/suspicious/noDuplicateObjectKeys)